tests/unit/test_audio_chunking.py

```
@@ -0,0 +1,275 @@
+        # File duration 10.1 seconds, chunk duration 10 seconds
+        assert self.chunker.should_chunk(10.1) is True
+
+    @patch('audio_separator.separator.audio_chunking.AudioSegment.from_file')
+    @patch('audio_separator.separator.audio_chunking.os.path.exists')
+    @patch('audio_separator.separator.audio_chunking.os.makedirs')
+    def test_split_audio_basic(self, mock_makedirs, mock_exists, mock_from_file):
+        """Test basic audio splitting."""
+        # Mock audio file (30 seconds)
+        mock_audio = Mock()
+        mock_audio.__len__ = Mock(return_value=30000)  # 30 seconds in ms
+        mock_audio.__getitem__ = Mock(side_effect=lambda key: mock_audio)
```

Ruff (0.14.11)

Unused lambda argument: `key`

(ARG005)